src/views/accounts/VirtualAccount/VirtualDetail/Detail.vue

```diff
@@ -30,7 +30,3 @@ export default {
   }
 }
 </script>
-
-<style scoped>
-
-</style>
```

Comment:
I'm sorry, but it's not clear what you're asking. The code that is being compared seems to be incomplete (missing some tags) and isn't formatted properly either. Could I ask you if there are specific issues related to the syntax or structure of these files?